doc/content.tex

```diff
@@ -1,1347 +0,0 @@
-past or analogously because there is a relationship between the future and the past. However, this relation is not
-deterministic and can be hardly written in an analytical form \cite{otexts}.
-
-Generally there are two forecasting types: qualitative and quantitative. Qualitative methods are mainly based on the
-opinion of the subject and are used when past data are not available, hence not suitable for this project. If there
-are past data available, quantitative forecasting methods are more suitable.
-
-    %%%%%%%%
-    \section{Simple Quantitative Models} \label{sec:simple-models}
-    Following methods are the simples forecasting quantitative models. They can be used on any time series without
-    any prerequisites or further analysis.
-
-    \begin{itemize}
-        \item Average method\,--\,forecasts are equal to the value of the mean of historical data.
-            \begin{eqnarray}
-                \hat{y}_{T+h|T} = \overline{y} = (y_{1}+ \dots + y_{T}) / T 
-            \end{eqnarray}
-        \item Na\"{i}ve method\,--\,forecasts are equal to the last observed value.
-            \begin{eqnarray}
-                \hat{y}_{T+h|T} = y_{T}
-            \end{eqnarray}
-        \item Drift method\,--\,variation of na\"{i}ve method which allow the forecasts to increase or decrease
-            over time.
-            \begin{eqnarray}
-                \hat{y}_{T+h|T} = y_{T} + \frac{h}{T-1} \sum_{t=2}^T{y_{t} - y_{t-1}} = 
-                    y_{T} + h(\frac{y_{T}-y_{1}}{T-1}) 
-            \end{eqnarray}
-    \end{itemize}
-
-    There are also seasonal variants of these models which holds one model for each season. These methods in general
-     produces high forecasting error but are very easy to implement.
-
-    %%%%%%%%
-    \section{Linear Regression} \label{sec:linear-regression}
-    Linear regression is a classical statistical analysis technique. It is mostly used to determine whether there is
-    linear relationship between dependent and eventually more independent variables \cite{cipra}. It is also used for
-    predictions mainly in econometric field.
-
-    Simple linear regression is defined as:
-
-    \begin{eqnarray} \label{eq:linear-regression}
-        y = \beta_0 + \beta_1 x + \epsilon
-    \end{eqnarray}
-
-    Parameters $\beta_0$ and $\beta_1$ are calculated by minimizing the sum of squared errors:
-    
-    \begin{eqnarray} \label{eq:linear-regression-estimation}
-        SSE = \sum_{i=1}^N \epsilon_{i}^2 = \sum_{i=1}^N (y_i - \beta_0 - \beta_1 x_i)^2
-    \end{eqnarray}
-    
-    Once the parameters are estimated predictions for any time in the future can be calculated. If modelled time
-    series is not stationary and trend changes over time parameters should be periodically reestimated to achieve
-    better forecasting accuracy. So the regression model does not adapt if the underlying time series changes.
```

Comment:
This means the regression model... ("So" by sa (podla starej skoly) nemalo velmi nachadzat na zaciatku vety)